### ISSUE TITLE
Weight unevenly distributed between scales

### ISSUE THREAD

**Issue by AgustinVallejo** [phetsims/buoyancy#132]
I noticed that if you put the two scales next to eachother and place the block in both of them, the pool scale will get like 75% of the block's weigth regardless how it's positioned. See images below, in both cases, the scale with the 15N is the one that started in the pool

Should we do something about this or are we good to ignore it since it's not part of the learning goals?

![image](https://github.com/phetsims/buoyancy/assets/37052186/804fda3f-6a6b-475b-842a-d5d7e705ab43)

![image](https://github.com/phetsims/buoyancy/assets/37052186/887f145a-6945-4439-8acd-4724dad9d3da)

**Comment by DianaTavares** [comment]
I cannot reproduce this bug. When I place the bock over the two scales, the readout only goes crazy. 
![scales](https://github.com/phetsims/buoyancy/assets/26031501/916fe71c-cde9-435c-9988-ba78c3bfa7de)

But that also still happening in explore with for example with aluminum:

![aluminium](https://github.com/phetsims/buoyancy/assets/26031501/8277367f-5b64-44c3-b64b-08609907fbeb)


- [ ] We need first to fix the scales to display a number that doesn't oscillate

And about the two scales, we can make it simple (because it is not the learning goal) and just divide them between 2, or we can use calculate the value with equilibrium equations. Knowing that the addition of the values of both scales need to be equal to the block's weight, with torques. In that case, you need to know the distance between the scale and the center of mass of the block. What do you think?

**Comment by samreid** [comment]
Let's discuss this at an upcoming standup

**Comment by samreid** [comment]
In today's standup, @AgustinVallejo and @DianaTavares and I reproduce the problem. We saw that the problem is worse for light materials like wood, we think it is because it is penetrating the scale surface less. We adjusted the physics engine like so:

<details>

```diff

Subject: [PATCH] Add REVIEW comments, see https://github.com/phetsims/density-buoyancy-common/issues/123
---
Index: js/common/DensityBuoyancyCommonQueryParameters.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/js/common/DensityBuoyancyCommonQueryParameters.ts b/js/common/DensityBuoyancyCommonQueryParameters.ts
--- a/js/common/DensityBuoyancyCommonQueryParameters.ts	(revision 7521809f7bb422f8363da10649cfdfe24611ae25)
+++ b/js/common/DensityBuoyancyCommonQueryParameters.ts	(date 1716999460056)
@@ -140,7 +140,7 @@
   // Stiffness to use for mass-mass interactions
   p2DynamicStiffness: {
     type: 'number',
-    defaultValue: 1e6
+    defaultValue: 1e2
   },
 
   // Relaxation:
@@ -163,7 +163,7 @@
   // Relaxation to use for mass-mass interactions
   p2DynamicRelaxation: {
     type: 'number',
-    defaultValue: 4
+    defaultValue: 8
   },
 
   // Controls the proportional amount of overall result viscosity forces applied

```

</details>

This solved it for wood:

<img width="405" alt="image" src="https://github.com/phetsims/buoyancy/assets/679486/a5f7fe3f-c207-405c-8745-20c0d2703081">

However, now there are problems for more dense materials, like this:

<img width="338" alt="image" src="https://github.com/phetsims/buoyancy/assets/679486/a822cf54-0055-4a6d-8af3-d4dc08eae9a9">

In conclusion, we agreed to close the issue.
